reaskapi/api_client.py

```diff
@@ -101,6 +101,10 @@ def _call_api(self, param_args, endpoint, method='GET', post_data={}):
         Base method to send authenticated calls to the API HTTP endpoints
         """
 
+        if 'product_version' in param_args:
```

Comment:
It might be ok for now, but in general, it would be better that `product_version` is a member of ApiClient and initialised at the beginning, i.e. `dc = DeepCyc(product_version = 'DeepCyc-2.0.6')` then copy `self.product_version` to `self.headers['product-version']` if it is not `None`.

Reply:
OK, good idea.